Doc/library/functions.rst

```diff
@@ -562,6 +562,11 @@ are always available.  They are listed here in alphabetical order.
       >>> eval('x+1')
       2
 
+   .. warning::
```

Comment:
The `exec` warning is nice and prominent right under the function signature:

https://cpython-previews--118437.org.readthedocs.build/en/118437/library/functions.html#exec

This `eval` warning is halfway down, after an example:

https://cpython-previews--118437.org.readthedocs.build/en/118437/library/functions.html#eval

Shall we move this one up, under the signature, or perhaps right after the parameters table?

Reply:
Good question. Unfortunately `exec` has no parameters table so it is a bit inconsistent.

What do the others think, where should the warning be?

Reply:
Personally I would like to see this warning under the parameters table for the ``eval`` and under the signature for the ``exec`` (since ``exec`` does not has parameter table).

Reply:
I've moved it now for `eval` as suggested (at least I think I did it correctly).